postgresql-ballerina/types.bal

```diff
@@ -42,6 +52,18 @@ public distinct class CidrValue {
     }  
 }
 
+# Represents Cidr array PostgreSQL Field
+#
+# + value - Value of parameter passed into the SQL statement
+public distinct class CidrArrayValue {
```

Comment:
@daneshk @niveathika Can we remove the suffix as `Value` from the Array type names?

```suggestion
public distinct class CidrArrayValue {
```

Reply:
+1

Reply:
I have changed. It looks good[1] but some array types[2] defined in the SQL module with the `Value` suffix. Can we do this change for those names?

[1] https://github.com/ballerina-platform/module-ballerinax-postgresql/pull/7/files#diff-a9e5a92ee7ba7bf543bf2c119aaa7db5c7ea3b8d4a6681ce558171dfd2f1e94cR1302
[2] https://github.com/ballerina-platform/module-ballerinax-postgresql/pull/7/files#diff-a9e5a92ee7ba7bf543bf2c119aaa7db5c7ea3b8d4a6681ce558171dfd2f1e94cR1183

Reply:
you have a valid point. we don't need the suffix `Value`, but let's keep it for consistency.